package.json

```diff
@@ -1,49 +1,53 @@
 {
   "name": "react-chat-renderer",
-  "version": "0.0.7",
+  "version": "0.1.0-alpha.12",
   "license": "MIT",
   "scripts": {
     "postversion": "npm publish",
-    "preversion": "npm test",
+    "preversion": "yarn test && yarn build",
```

Comment:
why not yarn build then test?

Reply:
cause build does tsc/babel transpile needed for npm packaging... test doesn't need that and if tests fail, there's no point in packaging.